app/models/manageiq/providers/ibm_cloud/inventory/persister/power_virtual_servers.rb

```diff
@@ -45,6 +45,16 @@ def initialize_cloud_inventory_collections
       builder.add_default_values(:ems_id => ->(persister) { persister.cloud_manager.id })
     end
 
+    add_collection(cloud, :system_types) { |builder|
+      builder.add_properties(:system_types => ::ManageIQ::Providers::IbmCloud::PowerVirtualServers::CloudManager::SystemTypes)
```

Comment:
I think what you're looking for here is `builder.add_properties(:type => ManageIQ::Providers::IbmCloud::PowerVirtualServers::CloudManager::SystemTypes)`